src/routes/safe/components/Transactions/helpers/utils.ts

```diff
@@ -16,3 +18,12 @@ export const areSafeParamsEnabled = (parametersStatus: ParametersStatus): boolea
 export const ethereumTxParametersTitle = (isExecution: boolean): string => {
   return `Owner transaction ${isExecution ? '(Execution)' : '(On-chain approval)'}`
 }
+
+export const getInteractionTitle = (value: number | string): string => {
+  if (value === 0 || (typeof value === 'string' && parseFloat(value) === 0)) {
+    return 'Interact with:'
+  }
+
+  const { symbol } = getNativeCurrency()
+  return `Interact with (and send ${value} ${symbol} to):`
```

Comment:
I know this was requested by @johannesmoormann but tbh, this wording feels a bit clumsy. Especially for simple transfers.
I would just keep the old one for this case.

Reply:
I was given the green light by @johannesmoormann this morning to change them all.